evalit/mft/mft_automation.py

```diff
@@ -48,13 +50,15 @@ def run_automation(self):
         dest_storage_id = "d602acab-595a-44ac-a9b3-60c5ceca80e7"
```

Comment:
is this supposed to be hardcoded? would it make more sense to put this in the env variable

Reply:
Will have to check with @smarru as I don't have much idea on this. This PR is a code cleanup. So there's definitely more room for optimization/improvements. Thanks for pointing out this one.